machines/Issuers/IssuersActions.ts

```
@@ -296,18 +296,18 @@ export const IssuersActions = (model: any) => {
     }),
     setCredentialOfferCredentialType: model.assign({
       selectedCredentialType: (context: any, event: any) => {
-        let credentialTypes: Array<{ id: string;[key: string]: any }> = [];
+        let credentialTypes: Array<{id: string; [key: string]: any}> = [];
```

ESLint

[error] 'credentialTypes' is never reassigned. Use 'const' instead.

(prefer-const)